georgia/covid-viz.R

```diff
@@ -7,7 +7,7 @@ library(zoo)
 raw_dat <- behindbarstools::read_scrape_data(all_dates = TRUE, state = "Georgia")
 ga_state <- raw_dat %>%
     filter(Jurisdiction == "state",
-           Age != "Juvenile") %>%
+           !Age %in% c("Juvenile")) %>%
```

Comment:
Original code dropped rows where `Age` is NA, new code keeps them. Went from 20,923 rows in `ga_state` to 21,467.

Reply:
Good catch, thank you!